webrepl.html

```diff
@@ -97,6 +97,10 @@
 
 (function() {
     window.onload = function() {
+      var url = window.location.hash.substring(1);
+      if(url) {
+      	document.getElementById('url').value = 'ws://' + url;
```

Comment:
The whole window.onload method is using 2 spaces, and so all indents should use 2 spaces, not 1, like patch above. Changing unrelated things is always the worst solution. Please leave it as is, maintainers will clean it all up (when have a time).